src/Symfony/Component/ErrorHandler/Resources/views/logs.html.php

```diff
@@ -23,7 +23,7 @@
             $status = \E_DEPRECATED === $severity || \E_USER_DEPRECATED === $severity ? 'warning' : 'normal';
         } ?>
         <tr class="status-<?= $status; ?>" data-filter-level="<?= strtolower($this->escape($log['priorityName'])); ?>"<?php if ($channelIsDefined) { ?> data-filter-channel="<?= $this->escape($log['channel']); ?>"<?php } ?>>
-            <td class="text-small" nowrap>
+            <td class="text-small" style="white-space:nowrap">
```

Comment:
should be replaced by a class, with styles defined in the CSS of the component instead.